sample-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,33 +1,36 @@
 object Versions {
-    const val BUILD_GRADLE_VERSION = "4.2.1"
+    const val BUILD_GRADLE_VERSION = "7.3.0"
 
-    const val ANDROID_COMPILE_SDK_VERSION = 30
-    const val ANDROID_MIN_SDK_VERSION = 23
-    const val ANDROID_TARGET_SDK_VERSION = 30
+    const val ANDROID_COMPILE_SDK_VERSION = 33
+    const val ANDROID_MIN_SDK_VERSION = 24
+    const val ANDROID_TARGET_SDK_VERSION = 33
 
     const val ANDROID_VERSION_CODE = 1
     const val ANDROID_VERSION_NAME = "3.14.0"
 
     // Dependencies (Alphabet sorted)
     const val ANDROID_COMMON_KTX_VERSION = "0.1.1"
     const val ANDROID_CRYPTO_VERSION = "1.0.0"
-    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.2.1"
-    const val ANDROIDX_CORE_KTX_VERSION = "1.3.0"
-    const val ANDROIDX_FRAGMENT_VERSION = "1.3.3"
-    const val ANDROIDX_LIFECYCLE_VERSION = "2.4.0-alpha02"
+    const val ANDROIDX_ACTIVITY_KTX_VERSION = "1.6.0"
```

Comment:
I think we can remove `androidx.activity:activity-ktx` in `build.gradle.kts` as we don't apply anything from it 🤔

Reply:
@Wadeewee @doannimble  In case it can be removed, do you think we should keep it for a different pull request?

This pull request is only meant to apply the changes from #323 to the samples 🙏 

Same goes for the suggestion below

Reply:
Agree @Tuubz

Reply:
> @Wadeewee @doannimble In case it can be removed, do you think we should keep it for a different pull request?

Yeah, I agreed @Tuubz 🚀